km52/Larionova_Yuliia/laboratory1.sql

```diff
@@ -1,2 +1,135 @@
 -- LABORATORY WORK 1
 -- BY Larionova_Yuliia
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+вставляти дані до таблиць
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+create user larionova identified by pass
+default tablespace "USERS"
+temporary tablespace "TEMP";    
+
+ALTER USER larionova QUOTA 100M ON USERS;
+
+GRANT "CONNECT" TO larionova ;
+
+GRANT INSERT ANY TABLE TO larionova ;
+
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Класна кімната має парти та стільці.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+create table chairs(
+    chair_name varchar2(40) not null
+);
+
+alter table chairs
+    ADD CONSTRAINT chairs_pk PRIMARY KEY (chair_name);
+
+
+create table desks(
+    desk_name varchar2(40) not null
```

Comment:
`desk_material varchar2(40) not null,`
`desk_description varchar2(40) not null,`
`desk_age number(10, 0) not null`